### ISSUE TITLE
oneOf(Object.keys(SIZES)) is propTypes

### ISSUE THREAD

**Issue by romanlex** [sapegin/react-styleguidist#986]
Hello
I see what supporting `Object.keys` in `react-docgen` is merged
https://github.com/reactjs/react-docgen/pull/211

When `react-styleguidist` add this support?

**Comment by sapegin** [comment]
As soon as someone sends a pull request with an update ;-)

**Comment by stepancar** [comment]
@romanlex , could you check, please?

**Comment by romanlex** [comment]
it's doesn't work(
```js
const COLORS = Object.keys(colors)
```
![image](https://user-images.githubusercontent.com/3369304/41837350-e3f228a2-7865-11e8-8c72-ef5f451b107d.png)

**Comment by andreapiras** [comment]
> It looks like the colors example works for me

@jmetev1 yeah that works as long as colors is defined inside the same file. If you import colors from external file (which is a very common pattern) doesn't work
